src/Components/SystemDetail/RecommendationsTable.scss

```diff
@@ -2,15 +2,43 @@
   tr {
     --pf-c-table--cell--FontSize: var(--pf-global--FontSize--md);
   }
-  .newline {
-    white-space: pre-line;
-  }
-  .tab {
-    white-space: pre-wrap;
-  }
   .recs-table-empty {
     .pf-c-empty-state__icon {
       fill: var(--pf-global--success-color--100);
     }
   }
 }
+.newline {
+  white-space: pre-line;
+}
+.tab {
+  white-space: pre-wrap;
+}
+.Th-tag-style {
```

Comment:
@r14chandra since we want it for all table headers I would suggest to revert it to be TH or if you want to use custom class can you please rename it to small case `th-tag-style` for consistent naming convention.